corehq/apps/userreports/sql/adapter.py

```diff
@@ -211,8 +211,13 @@ def bulk_delete(self, docs, use_shard_col=True):
         table = self.get_table()
         doc_ids = [doc['_id'] for doc in docs]
         delete = table.delete(table.c.doc_id.in_(doc_ids))
-        with self.session_context() as session:
-            session.execute(delete)
+        try:
+            with self.session_context() as session:
+                session.execute(delete)
+        except ProgrammingError as e:
+            if not self.table_exists:
+                return
+            raise e
```

Comment:
Not really worth the hassle of changing, but maybe just interesting to know about: You can use `raise` to raise the last exception, and so you don't need a variable for it in this situation.

```suggestion
        try:
            with self.session_context() as session:
                session.execute(delete)
        except ProgrammingError:
            if not self.table_exists:
                return
            raise
```

Reply:
Good point. I forgot about that :+1: